src/docs/operator-guides/features/index.md

```diff
@@ -373,6 +373,21 @@ one of the
 and everyone receiving money through your marketplace should be based in
 one of the [countries supported by Stripe](https://stripe.com/global).
 
+## Content management
+
+You can manage much of your marketplace content in Flex Console. In
```

Comment:
I wonder if "marketplace content" sounds like user-generated content. Should we clarify that? For example:

"You can manage much of your _static content_ (texts and images in the user interface of your marketplace) in Flex Console.

Reply:
Fair point, will make a change in that direction!

Reply:
Will likely still make some changes in this wording, cf conversation below about "static content" not being the correct term for pages populated with CMS dynamically.

Reply:
Modified the wording by skipping `content` entirely: 
> You can manage much of your marketplace texts and images in Flex Console.